### PR TITLE
Add note about calendar invites to templates

### PR DESCRIPTION
**Pull request by eddiezane**
This PR adds a note to all org templates about receiving calendar invites by joining the mailing list.

I've encountered many folks in the community (both new and old) that didn't know this was a thing. I think it makes sense to explicitly call it out.

Happy to change the wording.

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Comment by mrbobbytables** [comment]
/approve
/hold
just to get an lgtm from another root owner quick

**Comment by alisondy** [comment]
Heyo, would it be possible to reword to something like

`Joining the mailing list for the group will typically add the invites for the following meetings to your calendar. The mailing list details can be found in the Contact section`

and also link to the contact section in the doc where the mailing list details are?

**Comment by eddiezane** [comment]
@alisondy done!

Can also drop the last sentence and link directly to the mailing list if that's better.

**Comment by alisondy** [comment]
> @alisondy done!
> 
> Can also drop the last sentence and link directly to the mailing list if that's better.

dropping last sentence and linking to mailing list directly sounds good to me

**Comment by eddiezane** [comment]
@alisondy done

**Comment by brancz** [comment]
Should we extend this to the meeting notes docs, it typically follows the same as the calendar invites.

File: generator/committee_readme.tmpl
Comment (justaugustus) [inline generator/committee_readme.tmpl:13]:
```suggestion
## Meetings

{{- if .Contact.MailingList }}
*Joining the [mailing list]({{.Contact.MailingList}}) for the group will typically add invites for the following meetings to your calendar.*
{{- end }}

```

Adding some spacing to prevent a markdown lint warning.

**Review by justaugustus — CHANGES_REQUESTED** [review]
@eddiezane -- Nice addition! Would you mind addressing my review and then regenerating the docs?

**Comment by justaugustus** [comment]
@eddiezane -- Ping on https://github.com/kubernetes/community/pull/5373#pullrequestreview-801802471 :)

**Comment by eddiezane** [comment]
@justaugustus done!

I'm made your change to each template (the spacing is a bit tight in all of them) but I'm not sure it had the output you wanted =/

/assign @justaugustus

**Review by justaugustus — APPROVED** [review]
> I made your change to each template (the spacing is a bit tight in all of them) but I'm not sure it had the output you wanted =/

@eddiezane -- No worries! There are some tweaks here and there that would be needed to fix all of the markdown lint warnings, but it doesn't make sense to block on that here.

Thanks for this improvement to the READMEs!
/lgtm
/approve
/hold cancel

**Pull request merged**
